flw/src/main/java/org/motechproject/nms/flw/service/impl/FrontLineWorkerServiceImpl.java

```diff
@@ -93,9 +182,11 @@ public void delete(FrontLineWorker record) {
 
     @Override
     public void deleteAllowed(FrontLineWorker frontLineWorker) {
```

Comment:
The name is misleading, I would assume the this would return `true` if it's possible to delete the FLW.